### PR TITLE
some changes in my opinion

### PR DESCRIPTION
**Pull request by Bodengriller**
### Requirements

* I thought there woulkd be some improvements

### Description

edited readme and flash.sh
I think it is unnecessary to push avrdude-install at every execution of the script.
Additionally I think it is nice to be asked for the filename instead of using a fixed name.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
 **Artillery Sidewinder X1 Firmware Mod**  
    
 The ASWX1-FW-Mod is an optimization for the Artillery Sidewinder X1 3D printer.  
-The Artillery Sidewinder X1 is delivered with Marlin 1.19 [link](http://www.artillery3d.com/DownLoad/15688.html) and deactivated EEPROM memory function (M500).
```
Comment (pinguinpfleger) [inline README.md:5]: ok

File: README.md [hunk 2]
```diff
@@ -2,12 +2,12 @@
 **Artillery Sidewinder X1 Firmware Mod**  
    
 The ASWX1-FW-Mod is an optimization for the Artillery Sidewinder X1 3D printer.  
-The Artillery Sidewinder X1 is delivered with Marlin 1.19 [link](http://www.artillery3d.com/DownLoad/15688.html) and deactivated EEPROM memory function (M500).  
+The Artillery Sidewinder X1 is delivered with [Marlin 1.19](http://www.artillery3d.com/DownLoad/15688.html) and deactivated EEPROM memory function (M500).  
   
 This optimized firmware is based on [Marlin Firmware Version 2.0.x](https://github.com/MarlinFirmware/Marlin/tree/2.0.x)  
 and on Marlin [Artillery Sidewinder X1 config](https://github.com/MarlinFirmware/Configurations/tree/master/config/examples/Artillery/Sidewinder%20X1)  
 
-There is also an (optimized firmware for Artillery Sidewinder X1 touch display)[https://github.com/pinguinpfleger/ASWX1-TFTFW-MOD] which you can install too but it is optionally.
```
Comment (pinguinpfleger) [inline README.md:10]: already fixed

File: README.md [hunk 3]
```diff
@@ -25,66 +25,87 @@ There is also an (optimized firmware for Artillery Sidewinder X1 touch display)[
     No need for high acceleration and jerk values to get sharp edges.  
    Read https://marlinfw.org/docs/features/lin_advance.html for more details and how to calibrate.  
    By default the K_Factor is set to 0, so it is disabled.  
-   To enable it using gcode you should first calibrate your specific K factor.  
-   You can do this [here](https://marlinfw.org/tools/lin_advance/k-factor.html). Accordingly set the K factor within your slicer using e.g. `M900 K0.2`  
+   To enable it using gcode you should first [calibrate your specific K factor](https://marlinfw.org/tools/lin_advance/k-factor.html).
```
Comment (pinguinpfleger) [inline README.md:31]: ok

File: flash.sh [hunk 4]
```diff
@@ -1,18 +1,11 @@
...
-apt-get install avrdude
-# Mac
-#brew install avrdude
-
-PROGRAM='avrdude'
-
-# Change it to the firmware file path in hex format.
-FIRMWARE_HEX='./firmware.hex'
+# query filename
+echo "Enter filename of the *.hex to flash"
```
Comment (pinguinpfleger) [inline flash.sh]: I think it is kind of annoying if you always have to enter the filename.
The script also can't run unattended anymore.
Better solution would be to use command line argument and if not set take the default file name.

**Review by pinguinpfleger — COMMENTED** [review]
ich probier mal

File: flash.sh (hunk 4 above)
Comment (Bodengriller) [reply flash.sh]: great idea - let's take it on ToDo

**Review by pinguinpfleger — APPROVED** [review]

**Review by pinguinpfleger — APPROVED** [review]

**Pull request closed without merging**